File: official/docs/golang/current/smartrate/retrieve-standalone-smartrate-deliver-by.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func retrieveStandaloneSmartrateDeliverBy() {
	client := easypost.New("EASYPOST_API_KEY")

	params := &easypost.EstimateDeliveryDateForZipPairParams{
		FromZip:         "10001",
		ToZip:           "10002",
		Carriers:        []string{"UPS", "FedEx"},
		PlannedShipDate: "2024-07-18",
	}

	estimates, _ := client.EstimateDeliveryDateForZipPair(params)

	fmt.Println(estimates)
}

File: official/docs/python/v9/smartrate/retrieve-standalone-smartrate-deliver-by.py
import easypost

client = easypost.EasyPostClient("EASYPOST_API_KEY")

results = client.smartrate.estimate_delivery_date(
    from_zip="10001",
    to_zip="10002",
    planned_ship_date="2024-07-18",
    carriers=["usps"],
)

print(results)
